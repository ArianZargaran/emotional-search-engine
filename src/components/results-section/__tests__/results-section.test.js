import React from "react";
import ResultsSection from "../results-section";

it('renders ResultsSection correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<ResultsSection />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});