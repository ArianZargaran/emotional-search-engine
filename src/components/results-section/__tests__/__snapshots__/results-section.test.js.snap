// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ResultsSection correctly 1`] = `
<Context.Consumer>
  [Function]
</Context.Consumer>
`;
